api/pin.js

```diff
@@ -31,6 +32,7 @@ module.exports = async (req, res) => {
       icon_color,
       text_color,
       bg_color,
+      full_name,
```

Comment:
in here we would parse the value as boolean with `full_name: parseBoolean(full_name)`